share/completions/quilt.fish

```diff
@@ -1,35 +1,35 @@
+complete -c quilt -a grep -r --no-file -d "Grep through source files recursively (skips patches and quilt meta-information)"
+complete -c quilt -a header -r --no-file -d "Print/change the header of the topmost/given patch"
 complete -c quilt -a import -r -d "Import external patches"
-complete -c quilt -a mail -r -d "Create  mail  messages  from all patches in the series file, and either store or send them"
-complete -c quilt -a new -r -d "Create a new patch with the specified file name, and  insert  it after the topmost patch"
-complete -c quilt -a next -r -d "Print  the  name  of  the next patch after the specified or topmost patch in the series file"
-complete -c quilt -a patches -r -d "Print the list of patches that modify the specified file"
-complete -c quilt -a pop -r -d "Remove  patch(es) from the stack of applied  patches"
-complete -c quilt -a previous -r -d "Print the name of the previous patch before the specified or topmost patch in the series file"
+complete -c quilt -a mail -r -d "Create mail messages from patches in the series file, stores/sends them"
+complete -c quilt -a new -r -d "Creates new patch with the given name, inserts it after topmost patch"
+complete -c quilt -a next -r -d "Prints name of patch after the topmost/given patch in the series file"
+complete -c quilt -a patches -r -d "Print the list of patches that modify the given file"
+complete -c quilt -a pop -r -d "Remove patch(es) from the stack of applied patches"
+complete -c quilt -a previous -r -d "Prints name of patch before the topmost/given patch in the series file"
 complete -c quilt -a push -r -d "Apply patch(es) from the series file"
-complete -c quilt -a refresh -r -d "Refreshes the specified patch, or the topmost patch by default"
-complete -c quilt -a remove -r -d "Remove one or more files from the topmost or named patch"
-complete -c quilt -a rename -r -d "Rename the topmost or named patch"
+complete -c quilt -a refresh -r -d "Refreshes the given patch, or the topmost patch by default"
+complete -c quilt -a remove -r -d "Remove one or more files from the topmost/named patch"
+complete -c quilt -a rename -r -d "Rename the topmost/named patch"
 complete -c quilt -a series -r --no-file -d "Print the names of all patches in the series file"
-complete -c quilt -a setup -r -d " Initializes a source tree from an rpm spec file or a quilt series file"
-complete -c quilt -a snapshot -r --no-file -d " Take a snapshot of the current working state"
-complete -c quilt -a top -d "Print the name of the topmost patch on the current stack of applied patches"
-complete -c quilt -a unapplied -r -d "Print a list of patches that are not applied, or all patches that follow the specified patch in the series file"
-complete -c quilt -a upgrade -d "Upgrade the meta-data in a working tree from an old version of quilt to the current version"
+complete -c quilt -a setup -r -d "Initializes a source tree from an rpm spec file or a quilt series file"
```

Comment:
Maybe remove the first "file"? "Initialize a source tree from an rpm spec or quilt service file"?